src/vcpkg/commands.edit.cpp

```diff
@@ -40,11 +40,11 @@ namespace
 
         for (auto&& keypath : REGKEYS)
         {
-            const ExpectedL<std::string> code_installpath =
+            ExpectedL<std::string> code_installpath =
                 get_registry_string(keypath.root, keypath.subkey, "InstallLocation");
-            if (const auto c = code_installpath.get())
+            if (auto c = code_installpath.get())
             {
-                const Path install_path = *c;
+                Path install_path = std::move(*c);
                 output.push_back(install_path / "Code - Insiders.exe");
                 output.push_back(install_path / "Code.exe");
             }
```

Comment:
```
            if (const auto install_path = code_installpath.get())
            {
                output.push_back(*install_path / "Code - Insiders.exe");
                output.push_back(*install_path / "Code.exe");
            }
```

Reply:
Does not work because `get_registry_string` returns a `std::string` and output is a `std::vector<Path>`